File: src/Reference.CSharp/OpenStreetMapAddress.cs
using Newtonsoft.Json;

namespace Reference.CSharp
{
    /// <summary>
    /// OpenStreet map address
    /// </summary>
    public class OpenStreetMapAddress
    {
        /// <summary>
        /// City name
        /// </summary>
        [JsonProperty("city")]
        public string City { get; set; }
        /// <summary>
        /// County
        /// </summary>
        [JsonProperty("county")]
        public string County { get; set; }
        /// <summary>
        /// State
        /// </summary>
        [JsonProperty("state")]
        public string State { get; set; }
        /// <summary>
        /// Country
        /// </summary>
        [JsonProperty("country")]
        public string Country { get; set; }
        /// <summary>
        /// Country code
        /// </summary>
        [JsonProperty("country_code")]
        public string CountryCode { get; set; }
    }
}

File: src/Reference.VB/OpenStreetMapAddress.vb
Imports Newtonsoft.Json


''' <summary>
''' OpenStreet map address
''' </summary>
Public Class OpenStreetMapAddress
    ''' <summary>
    ''' City name
    ''' </summary>
    <JsonProperty("city")>
    Public Property City As String
    ''' <summary>
    ''' County
    ''' </summary>
    <JsonProperty("county")>
    Public Property County As String
    ''' <summary>
    ''' State
    ''' </summary>
    <JsonProperty("state")>
    Public Property State As String
    ''' <summary>
    ''' Country
    ''' </summary>
    <JsonProperty("country")>
    Public Property Country As String
    ''' <summary>
    ''' Country code
    ''' </summary>
    <JsonProperty("country_code")>
    Public Property CountryCode As String
End Class


